lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -7,7 +7,7 @@ module Exceptions
     # you must define the minor code in the locales file and call this class from
     # raise_error middleware.
     class BackendServiceException < BaseError
-      attr_reader :response_values, :original_status, :original_body
+      attr_reader :response_values, :original_status, :original_body, :key
```

Comment:
This was added so that `BackendServiceException` may be easily converted into it's service-specific form, [seen here](https://github.com/department-of-veterans-affairs/vets-api/pull/1905/files#diff-788ea7fbcaa41ab6482eb1f4d6eb3342R126)

Reply:
can you elaborate on this a bit? I'm not sure I understand. Though I think adding key as an attribute could be useful.

Reply:
I think it would work for initialization, but not work for reading it, but it's fine. doesn't hurt to make it accessible.